interfaces/builtin/unity7.go

```diff
@@ -639,7 +641,10 @@ func (iface *unity7Interface) AppArmorConnectedPlug(spec *apparmor.Specification
 	// but we don't care about that here because the rule above already
 	// does that) to '_'. Since we know that the desktop filename starts
 	// with the snap name, perform this conversion on the snap name.
-	// TODO parallel-install: use of proper instance/store name
+	//
+	// parallel-installs: UNITY_SNAP_NAME is used in the context of dbus
+	// mediation, this unintentionally opens access to dbus paths of keyed
+	// instances of @{SNAP_NAME} to @{SNAP_NAME} snap
```

Comment:
Nice catch. Please add an AppArmor policy comment above the affected rules. Eg:
```
# NOTE: when @{SNAP_NAME} == @{SNAP_INSTANCE_NAME}, this rule
# allows the snap to access parallel installs of this snap.
```

(we get policy audits from 3rd parties all the time and we need to note the deficiencies there).